Runtime/Cast/FixedLineCast.cs

```diff
@@ -35,9 +35,11 @@ protected override void GeneratePoints()
         {
             Vector3 originPosition = Origin.transform.position;
             Vector3 destinationPosition = originPosition + Origin.transform.forward * CurrentLength;
-
-            points[0] = originPosition;
-            points[1] = DestinationPointOverride != null ? (Vector3)DestinationPointOverride : destinationPosition;
+            if (points > 1)

```

Comment:
this isn't valid code, if you're trying to check for the length of the points List then you need to use `points.Count`

so:

```
if (points.Count > 1)
```